.gitignore

```diff
@@ -1 +1,2 @@
-ecom\
\ No newline at end of file
+ecom\
+.env\
```

Comment:
Please use the file name instead directory and ensure that .env file is not being exported